UI/index.html

```diff
@@ -0,0 +1,221 @@
+                    <button>Like</button>
+                </div>
+                <div class = "left">
+                    <p class="post-date"> On February 20, 2020 16:23 </p>
+                </div>
+            </div>
+        </div>
+
+        <div class="post">
+            <div class="icon"></div>
+            <span> Ariana G </span>
+            <p>Meditation is a great way to keep my body well-centered while juggling shooting schedules and recording sessions.</p>
+            <h>#meditation #join #health</h><br/>
+            <div class="post-footer">
+                <div class="right">
+                    <button>Like</button>
+                </div>
+                <div class = "left">
+                    <p class="post-date"> On February 20, 2020 9:37 </p>
+                </div>
+            </div>
+        </div>
+
+        <div class="post">
+            <div class="icon"></div>
+            <span> IU </span>
+            <p>I think I fell in love with Singapore.</p>
+            <h>#singapore #travelling </h><br/>
+            <div class="post-footer">
+                <div class="right">
+                    <button>Like</button>
+                </div>
+                <div class = "left">
+                    <p class="post-date"> On February 19, 2020 23:45 </p>
+                </div>
+            </div>
+        </div>
+
+        <div class="post">
+            <div class="icon"></div>
+            <span> Grif Hero </span>
+            <p>We are the heroes of our time!</p>
+            <h>#heroes #infinite </h><br/>
+            <div class="post-footer">
+                <div class="right">
+                    <button>Like</button>
+                </div>
+                <div class = "left">
+                    <p class="post-date"> On February 18, 2020 18:18 </p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <form>
+        <p><button class = "button-log">Log out</button></p>
+        <p><button class = "button-add">Add post</button></p>
+        <div class="filtering"> <h2> Filtering posts</h2>
+            <p>
+                <input type="checkbox" id="author"><label for="author">Authors</label><br/><br/>
+                <input type="text" placeholder="Input author"><br/>
+                <button class="add-filter"><ins>+Add more</ins></button><br/><br/>
+                <input type="checkbox" id="date"><label for="date">Date</label><br/><br/>
+                <input type="text" pattern="DD/MM/EEEE" placeholder="DD/MM/YYYY"><br/><br/>
+                <input type="checkbox" id="hashtag"><label for="hashtag">Hashtag</label><br/><br/>
+                <input type="text" placeholder="Input hashtag"><br/>
+                <button class="add-filter"><ins>+Add more</ins></button><br/><br/>
+            </p>
+        </div>
+    </form>
+    <form>
+        <p><button class = "button-load">Load more</button></p>
+    </form>
+</main>
+<footer>
+    <div id="footer">
```

Comment:
That div is not required since all the style could have been applied to \<footer\>